### DISCUSSION TITLE
Data nascita alunni

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#163] [Richieste di aiuto] [answered]
Buonasera,sarebbe possibile togliere non rendere la data di nascita degli alunni in ogni sezione, es.appello,voti etc?Grazie

**Comment by trinko** [comment, marked as answer]
La data di nascita è utilizzata per poter distinguere gli studenti omonimi.
Eventualmente si può modificare i template e nasconderle, ma francamente non ne vedo la necessità.
